io.podman_desktop.PodmanDesktop.yml

```diff
@@ -0,0 +1,98 @@
+app-id: io.podman_desktop.PodmanDesktop
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - "--socket=session-bus"
+  - "--socket=wayland"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--device=dri"
+  - "--filesystem=home"
+  - "--filesystem=xdg-run/podman"
+  - "--share=network"
+  - "--talk-name=org.freedesktop.Notifications"
+build-options:
+  append-path: /usr/lib/sdk/node16/bin
+modules:
+  # needs yarn to build the project
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - "cp -a * /app"
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: https://github.com/yarnpkg/yarn/releases/download/v1.22.19/yarn-v1.22.19.tar.gz
+        sha256: 732620bac8b1690d507274f025f3c6cfdc3627a84d9642e38a07452cc00e0f2e
+  # convert icon for all sizes
+  - name: ImageMagick
+    config-opts:
+      - "--disable-static"
+      - "--disable-docs"
+      - "--with-pic"
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/7.1.0-45.tar.gz
+        sha256: 3df6ca6dff15a4e8a20b4593c60285a59e38890440494d91a344e5c0e2bb3eec
+  # Podman Desktop sources
+  - name: podman-desktop
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/podman-desktop/flatpak-node/cache
+        YARN_CACHE_FOLDER: /run/build/podman-desktop/flatpak-node/yarn-cache
+    build-commands:
+      # Display tooling version
+      - node --version
+      - yarn --version
+      # configure mirror to use the one generated by generated-sources.json artifacts
+      - echo "yarn-offline-mirror $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror" > .yarnrc
+      # install in offline mode (all steps need to use offline with flatpak)
+      - yarn --offline install
+      # compile the app
+      - MODE=production yarn --offline run build
+      # Build electron app
+      - MODE=production $(npm bin)/electron-builder build --config .electron-builder.config.js --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # add icons
+      - for s in {32,64,128,256,512}; do
```

Comment:
you don't need this if you have a SVG.

Also rsvg-convert is in the SDK so you don't need to build ImageMagick either if you really insist on generating bitmaps.

Reply:
OK thanks, I will use rsvg-convert

Reply:
reworked using rsvg-convert